src/lib/util.ts

```diff
@@ -48,20 +49,23 @@ export const SUPPORTED_CHAINS: Record<ChainName, SupportedChain> = {
     chainId: 5,
     contract: proxies["ethereum-goerli"],
     host: "https://testnet.tableland.network",
+    rpcRelay: true,
```

Comment:
Added a default for each tableland network.
Trying to keep it simple for now, but when we add the mainet networks I'm thinking they would all have `rpcRelay: false` until we figure out tokenomics/approved-addresses/etc...  SDK users will be able to override this, but in the case of someone trying to relay on mainnet incorrectly the validator will return an error.